### PR TITLE
Allow integration tests to test commands that have confirmations

### PR DESCRIPTION
**Pull request by greg-1-anderson**
Pass --yes to confirm.

### CODE REVIEW

File: tests/unish/UnishIntegrationTestCase.php [hunk 1]
```diff
@@ -133,7 +133,7 @@ protected function buildCommandLine($command, $args, $options)
         }
         // insert drush command options
         foreach ($options as $key => $value) {
-            if (!isset($value)) {
+            if (!isset($value) || $value === true) {
```
Comment (greg-1-anderson) [inline tests/unish/UnishIntegrationTestCase.php]: Usually, `'yes' => true` gives `--yes`, and `'yes' => '1'` gives `--yes=1`. I've always found the Unish-style `'yes' => null` to be confusing. Would it be okay to switch to `true` everywhere?  We should also support `null` for external unish clients.

**Comment by greg-1-anderson** [comment]
Just one trivial question before merging. We can stick with `'yes' => null` if you feel strongly about it.

File: tests/unish/UnishIntegrationTestCase.php (hunk 1 above)
Comment (weitzman) [reply tests/unish/UnishIntegrationTestCase.php]: Yes feel free to change that.

File: tests/unish/UnishIntegrationTestCase.php [hunk 2]
```diff
@@ -102,7 +102,7 @@ public function drush($command, array $args = [], array $options = [], $expected
     protected function buildCommandLine($command, $args, $options)
     {
         $global_option_list = ['simulate', 'root', 'uri', 'include', 'config', 'alias-path', 'ssh-options', 'backend', 'cd'];
-        $options += ['root' => $this->webroot(), 'uri' => self::INTEGRATION_TEST_ENV]; // Default value.
+        $options += ['root' => $this->webroot(), 'uri' => self::INTEGRATION_TEST_ENV, 'strict' => 0]; // Default value.
```
Comment (weitzman) [inline tests/unish/UnishIntegrationTestCase.php]: Why disable strict mode?

File: tests/integration/WatchdogTest.php [hunk 3]
```diff
@@ -5,13 +5,15 @@
...
  */
-class WatchdogCase extends CommandUnishTestCase
+class WatchdogTest extends UnishIntegrationTestCase
 {
 
     public function testWatchdog()
     {
-        $this->setUpDrupal(1, true);
-        $this->drush('pm-enable', ['dblog']);
+        $this->drush('pm-enable', ['dblog'], [], self::IGNORE_EXIT_CODE);
```
Comment (weitzman) [inline tests/integration/WatchdogTest.php]: Why ignore exit code here?

File: tests/unish/UnishIntegrationTestCase.php (hunk 2 above)
Comment (greg-1-anderson) [reply tests/unish/UnishIntegrationTestCase.php]: Disabling strict mode bypasses argument validation. We can try without this.

File: tests/integration/WatchdogTest.php (hunk 3 above)
Comment (greg-1-anderson) [reply tests/integration/WatchdogTest.php]: We might enable modules again and again. I guess that enabling an already-enabled module is not an error, so we can remove this.

**Comment by greg-1-anderson** [comment]
It worked to remove strict mode; it was also fine to remove the ignore errors flag.

**Pull request merged**